documentation/modules/exploit/linux/local/ueb_priv_esc.md

```diff
@@ -0,0 +1,38 @@
+## Vulnerable Application
+
+  Unitrends UEB 9/10 local privesc
+
+  This exploit leverages bpserverd proprietary protocol to issue commands
+  as root.
```

Comment:
May want to add a section saying this was confirmed on 9.2 and 10.0
See https://github.com/rapid7/metasploit-framework/pull/10668/files#diff-1815cc92360882fded81d11cc48bb342R23 as a reference